helm/ako/values.yaml

```diff
@@ -36,7 +36,7 @@ AKOSettings:
   #   - kube-system
   #   - kube-public
   ipFamily: "" # This flag can take values V4 or V6 (default V4). This is for the backend pools to use ipv6 or ipv4. For frontside VS, use v6cidr
-
+  handleSecretsFromAKONSOnly: "false" # If the flag is switched on, handling of the secrets will be restricted only to the namespace where the AKO is installed.
```

Comment:
Can we re-word it as: `If this flag is set to true, AKO will handle secrets from the namespace where AKO is installed`

Reply:
I have addressed the comment.